### PR TITLE
Remove unnecessary classes in common.datatransfer package #6640

### LINKED ISSUE #6640: Remove unnecessary classes in common.datatransfer package
- `FeedbackQuestionBundle`: not used at all.
- `SessionResultsBundle`: an empty interface with only one implementing class. Removing it has no side effect.
- `FeedbackSessionState`: only used in `AdminSearchPageData`; can be moved there as a private enum instead.

### PR DESCRIPTION
**Pull request by twhite96**
Fixes #6640 

**Outline of Solution**
Added private enum from `FeedbackSessionState.java` class to `AdminSearchPageData.java` class, then deleted `FeedbackSessionState.java`. 

> Tell us how you solved the issue
Since the enum was already a part of the `FeedbackSessionState.java`, I really didn't have much coding to do. I copied the enum from `FeedbackSessionState.java` and pasted it into `AdminSearchPageData.java` as `FeedbackSessionState.java` class was unnecessary. I then deleted the feedback file and deleted the import class associated with it.

### CODE REVIEW

File: src/main/java/teammates/ui/pagedata/AdminSearchPageData.java [hunk 1]
```diff
@@ -23,6 +22,10 @@
 
 public class AdminSearchPageData extends PageData {
 
+    private enum FeedbackSessionState {
+      OPEN, CLOSED, PUBLISHED, AWAITING
```
Comment (samsontmr) [inline src/main/java/teammates/ui/pagedata/AdminSearchPageData.java]: Indentation should be 4 spaces, not 2.

**Comment by wkurniawan07** [comment]
Hi @twhite96, any updates for this PR?

**Comment by twhite96** [comment]
@wkurniawan07 @samsontmr is there anything else you want me to do? I have time. I will update the indentation but I am not sure what else I have permission to modify.

**Comment by samsontmr** [comment]
Unfortunately, this issue appears to have been resolved by another PR.

**Pull request closed without merging**